src/styles/header.scss

```diff
@@ -0,0 +1,64 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0 50px;
+  background-color: #fff;
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+
+    list-style: none;
+  }
+
+  &__item {
+    width: fit-content;
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    text-transform: uppercase;
+    font-weight: 500;
+    font-size: 12px;
+    display: block;
+    line-height: 60px;
+    color: #060b35;
+    text-decoration: none;
+    transition: all 300ms linear;
+
+    &:hover {
+      color: #00acdc;
+    }
+  }
+}
+
+a.is-active {
+  position: relative;
+  color: #00acdc;
+}
```

Comment:
In BEM methodology, it's not recommended to use tag names in selectors. It's better to use a class name instead of the tag name 'a'. For example, you can rename the class 'is-active' to 'nav__link--active' and apply it to the active link.